transcom/mymove
# B 20016 int

## [B-20016](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A949852)
 **GO 1.22.2 UPGRADE**
 
 https://dp3.atlassian.net/wiki/spaces/MT/pages/1931673649/How+to+Upgrade+Go+Version#3.-Update-transcom%2Fmymove-Repo

## Summary
Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author
-currently live in DEMO environment
These are to be checked by the author.

- [x] Tested in the DEMO environment (for changes to containers, app startup, or connection to data stores)

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access any app
2. Login as any user
3. make sure app doesn't break



### Backend


### Database

#### Any new migrations/schema changes:
